### ISSUE TITLE
Changing hyperparameter values in a single run

### ISSUE THREAD

**Issue by pratikborhade1** [hill-a/stable-baselines#549]
What will I have to do if I want to change a certain hyperparameter such as learning rate after regular intervals while training with an algorithm (Lets say PPO1) in a single run?

```

**System Info**
Describe the characteristic of your environment:
 * installed using pip
 * GTX 1060 3 gb i5-8500
 * Python version 3.6.9
 * Tensorflow version 1.11
 * Versions of any other relevant libraries

**Additional context**
Add any other context about the problem here.

**Comment by araffin** [comment]
Hello,

Please read the doc for that. in short:
- there is no unified way yet, on some you cannot, on others you may have to use callback
- for PPO2, SAC and TD3, you can pass a callable (check the doc and the rl zoo)

we should unify that when migrating to tf2 for the 3.0 release.
